### PR TITLE
[Rm-84612] Update libraries

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,10 +1,17 @@
+import com.github.jk1.license.filter.LicenseBundleNormalizer
+import com.github.jk1.license.render.JsonReportRenderer
+
 plugins {
     id 'java'
     id 'java-library'
     id 'maven-publish'
-    id "io.github.gradle-nexus.publish-plugin" version "1.1.0"
+    id "io.github.gradle-nexus.publish-plugin" version "1.3.0"
+    id "org.owasp.dependencycheck" version "9.0.9"
```
Comment (OptimumCode) [inline build.gradle:9]: I am not sure if you have seen this [release note](https://github.com/jeremylong/DependencyCheck#900-upgrade-notice) but I believe we must start migrating to NVD API and get the API key for our company

**Review by OptimumCode — APPROVED** [review]

**Comment by OptimumCode** [comment]
I also think we need to update workflows in the repository

File: build.gradle
Comment (OptimumCode) [inline build.gradle:1]: suggestion: if you are editing build.gradle anyway, why don't you migrate to th2 plugins as well?
The examples can be found [here](https://github.com/th2-net/th2-grpc-common/blob/dev-version-4/build.gradle) and [here](https://github.com/th2-net/th2-read-db/blob/dev-2/grpc/build.gradle.kts)

Comment (Nikita-Smirnov-Exactpro) [reply build.gradle:1]: @lumber1000 could you please look at this suggestion

**Comment by Nikita-Smirnov-Exactpro** [comment]
@lumber1000 could you please correct workflow to run manually in `dev release` and `release` modes

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+plugins {
+    id("java-library")
+    id("maven-publish")
+
+    val th2PluginVersion = "0.0.4"
+    id("com.exactpro.th2.gradle.base") version th2PluginVersion
```
Comment (OptimumCode) [inline build.gradle.kts:6]: note: gRPC th2 plugin automatically applies `base` plugins if it is applied to the root project. You can remove explicit application here

File: build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    id("com.exactpro.th2.gradle.grpc") version th2PluginVersion
+    id("com.exactpro.th2.gradle.publish") version th2PluginVersion
+}
+
+allprojects {
+    val version = project.properties["release_version"].toString()
+    val suffix = project.properties["version_suffix"]?.toString() ?: ""
+    this.group = "com.exactpro.th2"
+    this.version = version + if (suffix.isEmpty()) "" else "-$suffix"
+}
```
Comment (OptimumCode) [inline build.gradle.kts:16]: issue: We have only one project in this repo - why do we apply all this? If this is just a copy-paste problem kindly adapt the script for this project

File: build.gradle.kts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    id("com.exactpro.th2.gradle.base") version th2PluginVersion
+    id("com.exactpro.th2.gradle.grpc") version th2PluginVersion
+    id("com.exactpro.th2.gradle.publish") version th2PluginVersion
+}
+
+allprojects {
+    val version = project.properties["release_version"].toString()
+    val suffix = project.properties["version_suffix"]?.toString() ?: ""
+    this.group = "com.exactpro.th2"
+    this.version = version + if (suffix.isEmpty()) "" else "-$suffix"
```
Comment (OptimumCode) [inline build.gradle.kts:15]: issue: The `version_suffix` property won't be passed in the workflows you use. Probably, we need to remove this part as it does not do anything actually

File: build.gradle.kts (hunk 2 above)
Comment (lumber1000) [reply build.gradle.kts:6]: But this is the root project

Comment (OptimumCode) [reply build.gradle.kts:6]: Yes, and this is why you don't need to add `base` plugin manually. You can apply `grpc` and `publish` plugins only

**Review by OptimumCode — APPROVED** [review]

**Pull request merged**